### PR TITLE
SWAT gas mask for HoS

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Adds a SWAT gas mask. Replaces the standard issue Security gas mask in the HoC locker with an elite one.

**Media**
SWAT mask

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar
- add: Added the new elite SWAT gas mask.
- tweak: Replaced the standard issue Security gas mask in the HoS locker with a SWAT gas mask.

### CODE REVIEW

**Comment by deltanedas** [comment]
> The new elite gas mask SWAT

add is not included so use `Added the new elite SWAT gas mask.`

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/Clothing/Mask/swat.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e",
```
[the remark this answers is not in the document]
Comment (Nimfar11) [reply Resources/Textures/Clothing/Mask/swat.rsi/meta.json]: done

[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Textures/Clothing/Mask/swat.rsi/meta.json]: you just changed the hash, not linked to the .dmi or directory

[the remark this answers is not in the document]
Comment (Nimfar11) [reply Resources/Textures/Clothing/Mask/swat.rsi/meta.json]: Right?

[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Textures/Clothing/Mask/swat.rsi/meta.json]: theres a thing to link a file at a certain point in time, other rsis use it its like /blob/hash/path/to/file i think

[the remark this answers is not in the document]
Comment (Nimfar11) [reply Resources/Textures/Clothing/Mask/swat.rsi/meta.json]: Finally, I was able to find the original.

**Pull request merged**